public/video-ui/src/services/WorkflowApi.js

```diff
@@ -86,9 +85,9 @@ export default class WorkflowApi {
     video,
     status,
     section,
-    note
+    note,
+    prodOffice
   }) {
-    const prodOffice = getProductionOffice();
```

Comment:
This `getProductionOffice()` no longer required now the value is set manually.  It used to automatically set the production office based on timezone.

Reply:
See also deleted file below

Reply:
Might be worth checking this with Worth checking with Kirsten Broomhall as it means when we first track the atom in Workflow we have to explicitly set the production office rather than estimate it.

Reply:
Good point. This change will affect users..

Reply:
have emailled

Reply:
update: Kirsten wd like the default value of production office to be set to the office that we think they are in. Will reinstate the function and add a default in the select